src/main/java/uk/gov/justice/digital/delius/data/api/Conviction.java

```diff
@@ -10,7 +10,12 @@
 @Builder(toBuilder = true)
 public class Conviction {
     private Long convictionId;
+    private Long index;
```

Comment:
What does index mean?

Reply:
Good question; in Delius they use it as an identifier for an event in the UI. e.g the second Event added to Delius is known as the "event number 2". So it could be called the `offenderConvictionId` but I feel this might confuse any client in thinking it is some sort of unique id. On database they call it EVENT_NUMBER so I think index might be the best term, maybe `offenderIndex`